stories/samples/general.stories.js

```diff
@@ -42,7 +42,7 @@ export const Localization = () => html`
           noResultsFound: 'لم يتم العثور على نتائج',
           loadingMessage: 'Loading...'
         },
-        tasks: {
+        planner: {
```

Comment:
This is a breaking change and will require a docs PR. Let's make sure we note the breaking change in the list of changes.